### PR TITLE
Translated Advanced Algorithm Scripting in Italian

### LINKED ISSUE #9716: Translate "Advanced Algorithm Scripting" (advanced-bonfires.json) in Italian
#### Challenge Name
Advanced Algorithm Scripting ([link to challenge](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/advanced-bonfires.json))

#### Issue Description
Translate the challenge **Advanced Algorithm Scripting** in Italian

### PR DESCRIPTION
**Pull request by devilicecream**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [x] Add new translation (feature adding new translations)
#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

Italian translation for the challenge **Advanced Algorithm Scripting**

Closes #9716 - @Em-Ant hints implemented

### CODE REVIEW

**Comment by atjonathan** [comment]
@devilicecream, please ask someone in the Italian rooms to review your translation :smiley:

**Comment by devilicecream** [comment]
@atjonathan I did, and actually @Em-Ant has already reviewed this on my fork, where I created the first pull request by mistake. He or somebody else from the group will come to check it here as well anyway. 😄

**Comment by atjonathan** [comment]
@dakshshah96 awesome:tada:

**Comment by dakshshah96** [comment]
@atjonathan I guess you mean @devilicecream? 😛

**Comment by Em-Ant** [comment]
I've already reviewed the first version of this on @devilicecream 's fork, and we added some small improvement. I'm an Italian native speaker and  in my opinion this is a very good translation of an intricate technical subject. :+1:

**Comment by atjonathan** [comment]
We will merge for now and if something is incorrect, it can be changed later. Thanks for your PR @devilicecream :+1:

@dakshshah96 sorry :joy:

File: seed/challenges/01-front-end-development-certification/advanced-bonfires.json [hunk 1]
```diff
@@ -68,6 +68,14 @@
         "<blockquote>555-555-5555\n(555)555-5555\n(555) 555-5555\n555 555 5555\n5555555555\n1 555 555 5555</blockquote>",
         "Para esta prueba se te presentará una cadena de texto como por ejemplo: <code>800-692-7753</code> o <code>8oo-six427676;laskdjf</code>. Tu trabajo consiste en validar o rechazar el número telefónico tomando como base cualquier combinación de los formatos anteriormente presentados. El código de área es requrido. Si el código de país es provisto, debes confirmar que este es <code>1</code>. La función debe devolver true si la cadena de texto es un número telefónico válido en los EEUU; de lo contrario, debe devolver false.",
         "Recuerda utilizar <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/FreeCodeCamp-Get-Help' target='_blank'>Read-Search-Ask</a> si te sientes atascado. Intenta programar en pareja. Escribe tu propio código."
+      ],
+      "titleIt": "Verifica numeri telefonici degli Stati Uniti",
```
Comment (Em-Ant) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: It should be "Verifica **i** numeri telefonici degli Stati Uniti" (missing article).
I didn't notice it in the first review (sorry).

Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: @Em-Ant, can you make a pull request with the changes? Thanks

Comment (Em-Ant) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: @atjonathan done !

Comment (atjonathan) [inline seed/challenges/01-front-end-development-certification/advanced-bonfires.json]: Thanks :heart:

**Pull request merged**